### PR TITLE
feat: adds details about mask for text element

### PR DESCRIPTION
**Pull request by brigonzalez**
## Description

- adds details about mask for text element

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

Screen Shot 2021-11-22 at 10 38 58 AM

Screen Shot 2021-11-19 at 4 58 17 PM

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: source/includes/elements/_element_mask.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Element Mask
+
+```jsx
+var phoneNumberElement = BasisTheory.elements.create('text', {
+  targetId: 'myPhoneNumberElement',
+  mask: ['(', /\d/, /\d/, /\d/, ')', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]
+})
+```
+
+Text elements can restrict and fill user input by using the mask attribute. The mask attribute consists of an array
```
Comment (djejaquino) [inline source/includes/elements/_element_mask.md:10]:
```suggestion
Text elements can restrict and fill user input by using the `mask` attribute. It consists of an array
```

File: source/includes/elements/_element_mask.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+of `RegExp` objects and strings, where the `RegExp` objects are used to restrict input and the strings are used to fill input.
```
Comment (djejaquino) [inline source/includes/elements/_element_mask.md:11]:
```suggestion
of `RegExp` objects and strings, used to restrict and fill user input, respectively.
```

File: source/includes/elements/_element_mask.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+The position of the items in the mask array corresponds to the restriction or fill used for that input's position. The
```
Comment (djejaquino) [inline source/includes/elements/_element_mask.md:12]:
```suggestion
The position of each item in the mask array corresponds to the restriction or fill used for that input's position. The
```

**Review by djejaquino — CHANGES_REQUESTED** [review]
Good stuff. Just small verbiage suggestion

**Comment by armsteadj1** [comment]
The only update i'd add here is to let them know that the example in the first screenshot is a US based phone number.. for example (555)555-5555. Just so they know the context of the bullet point list. Looks great!!

**Review by djejaquino — APPROVED** [review]

**Pull request merged**